command/kv_delete.go

```diff
@@ -96,22 +113,55 @@ func (c *KVDeleteCommand) Run(args []string) int {
 		return 2
 	}
 
-	path := sanitizePath(args[0])
-	mountPath, v2, err := isKVv2(path, client)
-	if err != nil {
-		c.UI.Error(err.Error())
-		return 2
+	// If true, we're working with "-mount=secret foo" syntax.
+	// If false, we're using "secret/foo" syntax.
+	var mountFlagSyntax bool
+	if c.flagMount != "" {
+		mountFlagSyntax = true
+	}
+
+	var mountPath string
+	var partialPath string
+	var fullPath string
+	var v2 bool
```

Comment:
Style:
```suggestion
	var (
		mountPath   string
		partialPath string
		fullPath    string
 		v2          bool
 	)
```

Style: usually linters recommend grouping adjacent variables together.

Reply:
It may also be a good idea to move `fullPath`  declaration closer to where it's used (before the next block)